bazelrunner/src/main/java/org/jetbrains/bsp/bazel/bazelrunner/BazelRunner.kt

```diff
@@ -37,9 +37,7 @@ class BazelRunner private constructor(
   fun commandBuilder(): BazelRunnerCommandBuilder = BazelRunnerCommandBuilder(this)
 
   fun runBazelCommandBes(command: String, flags: List<String>, arguments: List<String>): BazelProcess {
-    fun besFlags() = listOf(
-        "--bes_backend=grpc://localhost:${besBackendPort!!}",
-        "--build_event_publish_all_actions")
+    fun besFlags() = listOf("--bes_backend=grpc://localhost:${besBackendPort!!}")
```

Comment:
dont we need this flag?

Reply:
we don't as far as I can tell.  According to the [bazel BEP reference](https://docs.bazel.build/versions/main/bep-glossary.html#actionexecuted) the flag only controls sending successful `ActionExecuted` events, but we only care about failed `ActionExecuted`, which are sent either way. https://github.com/JetBrains/bazel-bsp/blob/master/server/src/main/java/org/jetbrains/bsp/bazel/server/bep/BepServer.java#L137

Reply:
ohhhh, cool, my bad